### PR TITLE
Java Data Structures Review

### PR DESCRIPTION
**Pull request by edengarza**
- Many little changes for vocab clarity and easier code readability
- Fixed repl.it links so that they reflect the code in the right directory structure in our repl.it account
- Added a repl.it exercise for problem 1 of HashMaps

TODO
- [x] Unify the titles (i.e. sometimes a page is called examples vs. other times methods. Make them the same)
- [x] Add a repl.it exercise for problem 2 of HashMaps
    - Tricky because this was designed to be a capstone type, might reword the last page to not imply that
- [ ] Ensure that all repl.it activities can be successfully passed

### CODE REVIEW

File: content/english/java-datastructures/stacks/problem-1.md [hunk 1]
```diff
@@ -32,25 +32,17 @@ Oh no! A new intern was hired and messed up the list of restaurants! The inputs
...
-	restaurants.push("Fries");
-	restaurants.push("Pizza");
-	restaurants.push("Pizza");
+restaurants.push("Drink");
+restaurants.push("Fries");
+restaurants.push("Pizza");
+restaurants.push("Pizza");
 
-	//User ordered an extra pizza! how can you undo the recent change?
+// User ordered an extra pizza! how can you undo the recent change?
```
Comment (ozhang22) [inline content/english/java-datastructures/stacks/problem-1.md:43]:
```suggestion
// User ordered an extra pizza! How can you undo the recent change?
```

File: content/english/java-datastructures/stacks/methods.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+// Stack contains [2, 1]
+```
+
+<hr>
+
+With these three methods in mind, let's go ahead and take a look at an example.
+
+```js javascript
+import java.util.*;
+public class Driver{
```
Comment (ozhang22) [inline content/english/java-datastructures/stacks/methods.md:79]:
```suggestion
public class Driver {
```

File: content/english/java-datastructures/stacks/methods.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    public static void main(String []args){
```
Comment (ozhang22) [inline content/english/java-datastructures/stacks/methods.md:80]:
```suggestion
    public static void main(String []args) {
```

File: content/english/java-datastructures/2d-arrays/operations.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ public static void main(String[] args)
     arr[0][0] = 1;
```
Comment (ozhang22) [inline content/english/java-datastructures/2d-arrays/operations.md:19]: Can you fix this line to be consistent with how we use brackets?

```
public static void main(String[] args) { 
```

File: content/english/java-datastructures/2d-arrays/operations.md [hunk 5]
```diff
@@ -29,8 +29,14 @@ public static void main(String[] args)
 {
```
Comment (ozhang22) [inline content/english/java-datastructures/2d-arrays/operations.md:29]: Can you fix this line to be consistent with how we use brackets?

```
public static void main(String[] args) { 
```

File: content/english/java-datastructures/queues/methods.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+With these three methods in mind, let's go ahead and take a look at an example.
+
+```js javascript
+import java.util.*;
+ 
+class PriorityQueueDemo {
+   
+      // Main Method
+    public static void main(String args[])
+    {
```
Comment (ozhang22) [inline content/english/java-datastructures/queues/methods.md:86]:
```suggestion
    public static void main(String args[]) {
```

File: content/english/java-datastructures/queues/methods.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+Now our implementation should be solid.
+
+```js javascript
+import java.util.*;
+ 
+class PriorityQueueDemo {
+   
+      // Main Method
+    public static void main(String args[])
+    {
```
Comment (ozhang22) [inline content/english/java-datastructures/queues/methods.md:138]:
```suggestion
    public static void main(String args[]) {
```

**Review by ozhang22 — APPROVED** [review]

File: content/english/java-datastructures/2d-arrays/operations.md (hunk 5 above)
Comment (edengarza) [reply content/english/java-datastructures/2d-arrays/operations.md]: both from this page should be resolved now.

File: content/english/java-datastructures/arraylists/problem-1.md [hunk 8]
```diff
@@ -14,71 +14,71 @@ weight: 3
...
+menu.add("Hamburger"); 
+menu.add("Pizza");
+menu.add("Pizza");
+menu.add("Hotdog"); 
+
+// If we run the code
+item = remove(menu,"Pizza", 2);
+
+// remove() will remove the first two Pizza orders
+menu = {"Hotdog", "Hamburger", "Pizza", "Hotdog"}
```
Comment (ozhang22) [inline content/english/java-datastructures/arraylists/problem-1.md:79]:
```suggestion
// Remove the first two Pizza orders
item = remove(menu,"Pizza", 2);

// menu will now be the following array: {"Hotdog", "Hamburger", "Pizza", "Hotdog"}
```

**Review by ozhang22 — APPROVED** [review]
Looks good!

**Pull request merged**